Parsers/ASimWebSession/Test/Cisco_Firepower_ASimWebSession_DataTest.csv

```diff
@@ -0,0 +1,12 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 21 records (100.0%) for field [EventProduct] of type [Enumerated]: [""Firepower""] (Schema:WebSession)"
```

Comment:
Please update EventProduct and EventVendor in tester csv

Reply:
Done